### PR TITLE
Qt3 support warnings (other files) group 7

### PR DESCRIPTION
**Pull request by raquelalvarezbanos**
This PR fixes Qt3 support warnings in files:

Note.h
OpenProjectDialog.cpp
PeaksTabWidget.cpp
PeaksWorkspaceWidget.cpp
Plot.cpp
Plot3DDialog.cpp
PlotCurve.cpp
PlotDialog.cpp

**To test**

The best way to test this is testing each file individually, although I didn't know how to test some of them.
1. `Note`: not sure how to test this.
2. `OpenProjectDialog`: File > Load Project. Check that the window title is correct.
3. PeaksTabWidget: You'll need to build Vates. Alternatively, make sure that the change I made is the correct fix.
4. `PeaksWorkspaceWidget`: Load a MD dataset into the SliceViewer. I run the script `CreateSXDNaClQLab.py` which can be found in the `TrainingCourseData`. Run the script and load the MD dataset into the SliceViewer. You should be able to change the colors for the different types of peaks (cross, ellipsoid, spherical).
5. `Plot`: change is in a method that calculates the nearest data point to a point on the canvas. To test this I loaded and plotted a dataset, and clicked on the icon to add a new label. Click near a data point, you should get the correct (X,Y) coordinates.
6. `Plot3DDialog`: Load a dataset and open the matrix view. Choose one of the 3D plots available from the menu "3D Plot". Double click on one of the axis, a new window will appear. Test the different tabs and options, they should behave as expected.
7. `PlotCurve`: not sure how to test this, make sure that changes are OK.
8. `PlotDialog`: load a dataset and plot a spectrum. Right click on canvas and select "Properties". Test all the options (make sure you test both "Layer 1" and the options associated with the dataset).

Group 7 of #16129 

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: MantidPlot/src/Plot3DDialog.cpp [hunk 1]
```diff
@@ -783,7 +783,7 @@ bool Plot3DDialog::updatePlot()
 
 		try {
 			MyParser parser;
-			parser.SetExpr(to.ascii());
+			parser.SetExpr(to.latin1());
```
Comment (shahrozahmed1) [inline MantidPlot/src/Plot3DDialog.cpp]: Is there a reason you are using `.latin1`? shouldn't this be be `toAscii()` or `.toAscii().constData()` as you can find [here](http://doc.qt.io/qt-4.8/qstring-compat.html#ascii) ..

Comment (raquelalvarezbanos) [inline MantidPlot/src/Plot3DDialog.cpp]: The reason is that I started using `latin1` before realizing the right option is `.toAscii().constData()`. I replaced all the occurrences but apparently I missed some... Thanks!

File: MantidPlot/src/Plot3DDialog.cpp [hunk 2]
```diff
@@ -758,22 +758,22 @@ bool Plot3DDialog::updatePlot()
...
 		int axis = axesList->currentRow();
-		QString from=boxFrom->text().lower();
-		QString to=boxTo->text().lower();
+		QString from=boxFrom->text().toLower();
+		QString to=boxTo->text().toLower();
 		double start, end;
 		try {
 			MyParser parser;
-			parser.SetExpr(from.ascii());
+			parser.SetExpr(from.latin1());
```
Comment (shahrozahmed1) [inline MantidPlot/src/Plot3DDialog.cpp]: There is another one here too @raquelalvarezbanos

Comment (raquelalvarezbanos) [inline MantidPlot/src/Plot3DDialog.cpp]: Done! Thanks

**Comment by shahrozahmed1** [comment]
everything looks fine here, functional testing went as expected. Unable the test the one change under `Vates/` however the correct update has been made which can be found [here](http://doc.qt.io/qt-4.8/qtabwidget-compat.html#label). :shipit:

**Pull request merged**